platform/security/platform-security-core-api/src/main/java/org/codice/ddf/security/util/ThreadContextUtils.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.security.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import org.apache.shiro.util.ThreadContext;
+
+public final class ThreadContextUtils {
```

Comment:
✏️ This could be `ThreadContextProperties` or something like that to make it more descriptive what this class is doing instead of `Util`.